### PR TITLE
systemctl: add Spanish translation

### PR DESCRIPTION
**Pull request by maegop**
- [ ] The page (if new), does not already exist in the repository.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/linux/systemctl.cmd [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+- Mostrar todos los servicios en ejecución:
+
+`systemctl status`
+
+- Listar las unidades fallidas:
+
+`systemctl --failed`
+
+- Iniciar/Parar/Reiniciar/Recargar un servicio:
```
Comment (navarroaxel) [inline pages.es/linux/systemctl.cmd:14]:
```suggestion
- Inicia/Para/Reinicia/Recarga un servicio:
```

File: pages.es/linux/systemctl.cmd [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`systemctl {{start|stop|restart|reload}} {{unidad}}`
+
+- Mostrar el estado de una unidad:
```
Comment (navarroaxel) [inline pages.es/linux/systemctl.cmd:18]:
```suggestion
- Muestra el estado de una unidad:
```

File: pages.es/linux/systemctl.cmd [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+`systemctl status {{unidad}}`
+
+- Habilitar/Deshabilitar una unidad para que se inicie en el arranque:
```
Comment (navarroaxel) [inline pages.es/linux/systemctl.cmd:22]:
```suggestion
- Habilita/Deshabilita una unidad para que se inicie en el arranque:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Remember that for translations you can send more than 1 translated page in 1 PR, just keep 1 commit per file and I recommend no more than 15 pages. This doesn't apply to the main «English» pages.

**Review by CleanMachine1 — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**